MapboxNavigation/RouteViewController.swift

```diff
@@ -109,12 +109,65 @@ public class RouteViewController: NavigationPulleyViewController {
                    "or a storyboard reference to Navigation if you are using storyboards.")
     }
     
+    /**
+     Initializes a `RouteViewController` that provides turn by turn navigation
+     for the given route. A optional `direction` object is needed for  potential
+     rerouting.
+
+     See [MapboxDirections.swift](https://github.com/mapbox/MapboxDirections.swift)
+     for further information.
+     */
+    @objc(initWithRoute:directions:)
+    required public init(for route: Route,  directions: Directions = Directions.shared) {
+        let storyboard = UIStoryboard(name: "Navigation", bundle: Bundle.navigationUI)
+        let mapViewController = storyboard.instantiateViewController(withIdentifier: "RouteMapViewController") as! RouteMapViewController
+        let tableViewController = storyboard.instantiateViewController(withIdentifier: "RouteTableViewController") as! RouteTableViewController
+        
+        super.init(contentViewController: mapViewController, drawerViewController: tableViewController)
+        
+        ({
```

Comment:
I agree that it looks funny but it has to be on a separate line or it will trigger a compile-time warning.
It's not tied to super init call but it ensures that the code in the block is executed before the instance is returned.

This line https://github.com/mapbox/mapbox-navigation-ios/blob/fred-init/Examples/Swift/ViewController.swift#L160 crashes trying to access the implicitly unwrapped optional `routeController` as it's not initialized w/o this solution^